src/main/java/org/apache/accumulo/testing/continuous/ContinuousEnv.java

```diff
@@ -27,7 +27,7 @@ public ContinuousEnv(String[] args) {
         authList = Collections.singletonList(Authorizations.EMPTY);
       } else {
         authList = new ArrayList<>();
-        for (String a : authValue.split("|")) {
+        for (String a : authValue.split("\\|")) {
```

Comment:
If the double-escaping ever gets annoying, you can also use the square bracket syntax to create a character class, rather than use the escape character, as in: `.split("[|]")`